packages/editor/src/components/browser/functionBrowser/FunctionBrowser.tsx

```diff
@@ -1,159 +1,145 @@
+  const [type, setType] = useState('');
+
+  const { data: tree } = useMeta('meta/scripting/functions', undefined, []);
+  const { data: doc } = useMeta('meta/scripting/apiDoc', { context, method, paramTypes, type }, '');
+
+  const [selectedFunctionDoc, setSelectedFunctionDoc] = useState('');
+
+  const [showHelper, setShowHelper] = useState<boolean>(false);
+
+  const columns = useMemo<ColumnDef<Function>[]>(
+    () => [
+      {
+        accessorFn: row =>
+          `${row.name.split('.').pop()}${
+            row.isField === false ? `(${row.params.map(param => param.type.split('.').pop()).join(', ')})` : ''
+          }`,
+        id: 'name',
+        cell: cell => {
+          return (
+            <ExpandableCell
+              cell={cell}
+              title={cell.row.original.name}
+              icon={cell.row.original.isField ? IvyIcons.FolderOpen : IvyIcons.Function}
+              additionalInfo={cell.row.original.returnType ? cell.row.original.returnType.simpleName : 'no return type defined'}
+            />
+          );
+        }
+      }
+    ],
+    []
+  );
+
+  const [expanded, setExpanded] = useState<ExpandedState>(false as ExpandedState);
+  const [globalFilter, setGlobalFilter] = useState('');
+  const [rowSelection, setRowSelection] = useState<RowSelectionState>({});
+
+  const table = useReactTable({
+    data: tree,
+    columns: columns,
+    state: {
+      expanded,
+      globalFilter,
+      rowSelection
+    },
+    filterFromLeafRows: true,
+    enableRowSelection: true,
+    enableMultiRowSelection: false,
+    enableSubRowSelection: false,
+    onExpandedChange: setExpanded,
+    onGlobalFilterChange: setGlobalFilter,
+    onRowSelectionChange: setRowSelection,
+    getSubRows: row => (row.returnType ? row.returnType.functions : []),
+    getCoreRowModel: getCoreRowModel(),
+    getExpandedRowModel: getExpandedRowModel(),
+    getFilteredRowModel: getFilteredRowModel()
+  });
+
+  useEffect(() => {
+    if (Object.keys(rowSelection).length !== 1) {
+      setShowHelper(false);
+      props.onChange({ cursorValue: '' });
+      return;
+    }
+    const selectedRow = table.getRowModel().rowsById[Object.keys(rowSelection)[0]];
+    //setup correct functionName for the accept-method
+    const parentNames = getParentNames(selectedRow);
+    const selectedName = parentNames.reverse().join('.');
+    props.onChange({ cursorValue: selectedName });
+
+    //setup Meta-Call for docApi
+    const parentRow = selectedRow.getParentRow();
+    setType(
+      parentRow
+        ? parentRow.original.returnType.packageName + '.' + parentRow.original.returnType.simpleName
+        : selectedRow.original.returnType.packageName + '.' + selectedRow.original.returnType.simpleName
+    );
+    setMethod(selectedRow.original.name);
+    setParamTypes(selectedRow.original.params.map(param => param.type));
+    //setup Helpertext
+    setSelectedFunctionDoc(doc);
+    setShowHelper(true);
+  }, [doc, props, rowSelection, table]);
```

Comment:
Oh, I think because I moved each browser to its own folder, it no longer shows the individual changes. But actually, I only removed the getParentNames method here in the function browser, as I have outsourced it to the functionBrowser-utils.

Reply:
Don't understand what you mean here...